### PR TITLE
Seeding

### PR DESCRIPTION
**Pull request by tbright7**

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    connection.query(query, function (err, results) {
+      if (err) {
+        console.log(err)
+      }
+      callback(err, results);
+    });
+  }
+
+  const postAllProducts = function (product , callback) {
+      var query = `INSERT INTO products (title, price, description, category, image) VALUES ('${product.title}', ${product.price}, '${product.description}', '${product.category}', '${product.image}')`
```
Comment (loganrosenlund37) [inline database/index.js:17]: Would it be better to put the values being inserted into an array and using a sql string for the query?

**Review by loganrosenlund37 — APPROVED** [review]
Everything is looking good! I did have a comment about inserting into your database but works the way it is.

File: database/config.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
```
Comment (ecetino) [inline database/config.js:1]: It would be a good idea to put theses in a git ignore so your passwords are not available.

**Review by ecetino — APPROVED** [review]
Files look good. Code feels efficient.

**Pull request merged**